build/Microsoft.DotNet.Cli.DependencyVersions.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <CLI_MSBuild_Version>15.1.0-preview-000458-02</CLI_MSBuild_Version>
-    <CLI_NETSDK_Version>1.0.0-alpha-20161230-1</CLI_NETSDK_Version>
-    <CLI_WEBSDK_Version>1.0.0-alpha-20161205-1-154</CLI_WEBSDK_Version>
+    <CLI_NETSDK_Version>1.0.0-alpha-20170104-3</CLI_NETSDK_Version>
```

Comment:
Any chance you could bump this to `1.0.0-alpha-20170104-6` or later?  That way my fix for netcoreapp1.1 is included.